.github/PULL_REQUEST_TEMPLATE/new-pep.md

```diff
@@ -0,0 +1,72 @@
+---
+name: New PEP
+about: Submit a new draft PEP for consideration
+labels: "new-pep"
+---
+
+<!--
+
+Please read and follow PEP 1 and PEP 12 before submitting a new PEP:
+
+https://peps.python.org/pep-0001/#submitting-a-pep
+
+https://peps.python.org/pep-0012/#how-to-use-this-template
+
+Make sure to include the PEP number in the pull request title; for example:
+
+PEP 9999: <Title of PEP>
+
+See our Contributing Guidelines (CONTRIBUTING.rst) for more information.
+
+Thanks!
+
+-->
+
+
+# New PEP
+
+<!--
+
+Briefly describe your new PEP here, and anything else we should know.
+Be sure to link to any previous discussions or issues motivating/drafting it.
+
+-->
+
+
+
+## Checklist
+
+<!--
+
+You should use the following as a guide when double-checking your PEP,
+and you can help complete some of it yourself if you like
+by ticking any boxes you're sure about, like this: [x]
+
+If you're unsure about anything, just leave it blank and we'll take a look.
+
+-->
+
+* [ ] File created from the [latest PEP template](https://github.com/python/peps/blob/main/pep-0012/pep-NNNN.rst?plain=1)
+
+### Content requirements
+
+* [ ] PEP topic [discussed in a suitable venue](https://peps.python.org/pep-0001/#start-with-an-idea-for-python) and general agreement that a PEP is appropriate
+* [ ] Content is reasonably sound, complete and makes at least basic technical sense
+* [ ] Abstract and Copyright sections completed, with the required wording in the latter
+* [ ] [Suggested sections](https://peps.python.org/pep-0012/#suggested-sections) included (unless not applicable)
+* [ ] Title clearly, accurately and concisely describes the content
+* [ ] Prose is intelligible and generally free of basic grammar and syntax errors, at least to the degree they significantly detract from understanding
+* [ ] Code is in code blocks, well-formatted (PEP 7/PEP 8) and has the right [lexer name](https://pygments.org/docs/lexers/) if non-Python (or ``text``)
+* [ ] Any project stated in the PEP as supporting/endorsing benefiting from it confirms such
+
+
+### Technical requirements
+
+* [ ] Filename in the correct format (``pep-NNNN.rst``)
+* [ ] ``PEP``, ``Title``, ``Author``, ``Status`` (``Draft``), ``Type`` and ``Created`` headers filled out correctly
+* [ ] ``PEP-Delegate``, ``Requires``, ``Python-Version`` and ``Replaces`` filled out if appropriate
+* [ ] ``Sponsor`` listed and approves if not authored by a core dev or PEP editor
+* [ ] PEP builds with no warnings, CI checks pass and content displays as intended in the rendered preview
```

Comment:
This is hard to check ex-ante, given the preview only starts when the PR is opened.

Reply:
Well, they can always build it locally, check the preview and run the linting before they submit their final PR—that's fully equivalent to the CI. And as mentioned completion any item of the checklist prior to PR submission is entirely optional (though suggested).

Reply:
I revisied this to change "CI checks" to "pre-commit checks" and "rendered preview" to "rendered HTML", to make it apply equally to local and CI builds.

Reply:
Marking this as addressed; feel free to re-open or add another comment with any further feedback.